pages/common/pulumi-install.md

```diff
@@ -0,0 +1,12 @@
+# pulumi install
+
+> Install packages and plugins for the current program or policy pack.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_install>.
+
```

Comment:
```suggestion

- Install packages and plugins:

`pulumi install`

```
`pulumi install` works as-is, so I think we should have the base command listed.

Reply:
Looks good to me aside from this suggestion though!